.travis.yml

```diff
@@ -38,6 +52,20 @@ matrix:
       compiler: clang
       env: [CONFIGURATION=Release, *gcc-warnings]
 
+  # Sporadic failure builds
+    - compiler: gcc
+      env: CONFIGURATION=Debug    CTEST_OPTS="--repeat-until-fail 25 --schedule-random"
+      install: *install_cmake35
+      cache: *cache_homedir
+    - compiler: gcc
+      env: CONFIGURATION=Release  CTEST_OPTS="--repeat-until-fail 25 --schedule-random"
+      install: *install_cmake35
+      cache: *cache_homedir
+
+  allow_failures:
+    - env: CONFIGURATION=Debug    CTEST_OPTS="--repeat-until-fail 25 --schedule-random"
+    - env: CONFIGURATION=Release  CTEST_OPTS="--repeat-until-fail 25 --schedule-random"
+
```

Comment:
There must be a better way to constrain `env` and pass parameters to `cmake` and `ctest`; any ideas?